examples/pipelines/server-deployer/server-deployer-pipeline.yaml

```diff
@@ -64,7 +64,7 @@ spec:
               labels:
                 app: flasker-vm
             spec:
-              running: true
+              runStrategy: Halted
```

Comment:
Isn't `running: true` needs to be converted to `runStrategy: Always` instead of `Halted`?

Reply:
Yes. This was a result of pressue of end of the day :smile:

Reply:
I feel you :)